lib/client/workflow-client/workflowClient.js

```diff
@@ -155,7 +155,7 @@ WorkflowMediatorService.prototype.checkStatus = function checkStatus(workorder,
     status = CONSTANTS.STATUS.COMPLETE_DISPLAY;
   } else if (!workorder.assignee) {
     status = CONSTANTS.STATUS.UNASSIGNED_DISPLAY;
-  } else if (stepReview.nextStepIndex < 0) {
+  } else if (!result) {
```

Comment:
Basically we have here 3 different options that may indicate status:
- indexOfTheNextStep
- first/last step complete boolean flag
- result 

Why we picked result. Can we relay on first `firstStep.complete` as in if statement above?